### DISCUSSION TITLE
Welcome to zipline Discussions!

### DISCUSSION THREAD

**Discussion by JakeWharton** [cashapp/zipline#464] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MrWarren99** [comment]
Accessibility Business Developer

**Comment by jayteealao** [comment]
Wanted to try out zipline after watching your talk at droidcon and kotlinconf talk. However while trying to start the development server, i keep running into the same error. So i searched github for a sample project.
Running the sample project produces the same error, Im obviously doing something wrong but its hard to figure out what

Execution failed for task ':leaky-flow-js:compileDevelopmentExecutableKotlinJsZipline'.
> Could not initialize class app.cash.zipline.QuickJs

**Reply by JakeWharton** [reply]
If you're running locally from within the repo you need to build the native library first. The easiest way is to run either `.github/workflows/build-mac.sh` or `.github/workflows/build-linux-amd64.sh`.

**Reply by jayteealao** [reply]
I'm using release builds in my repo, I've tried versions 0.9.20, 19 and 13.
The sample app I tried is https://github.com/damianw/zipline-leaky-flow. But that failed as well.

**Reply by JakeWharton** [reply]
On what platform? If you're on Windows we currently don't support it.

**Reply by jayteealao** [reply]
That solves my problem. Thanks
Any workarounds?

**Reply by JakeWharton** [reply]
The JS engine we use does not build on Windows out of the box, which is why we don't support it. There are some patches on their mailing list and GitHub mirror that seem to enable it, but right now we're primarily focused on Android and iOS as the targets we want to support.

**Reply by Shabinder** [reply]
> A Makefile is provided to compile the engine on Linux or MacOS/X. A preliminary Windows support is available thru cross compilation on a Linux host with the MingGW tools.


confirming that zipline not supporting windows is still blocked on quickJS or is that not the case anymore and zipline in a future release can support it ?
